app/auth/google_auth.py

```
@@ -0,0 +1,73 @@
+        # 4️⃣ JWT 토큰 생성
+        access_token = create_access_token({"user_id": user.id})
+        refresh_token = create_refresh_token({"user_id": user.id})
+
+        return {
+            "user": user,
+            "access_token": access_token,
+            "refresh_token": refresh_token,
+        }
+
+    except Exception as e:
+        raise Exception(f"Google OAuth error: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)